Unity/Assets/SceneBuilders/MainSceneBuilder.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+using Zenject;
+
+public class MainSceneBuilder : MonoInstaller
+{
+    [SerializeField] GameObject mainCamera;
```

Comment:
Need a private after the [serializeField], as mentioned here - https://github.com/AdolfoPSakata/UnityTraining-2/pull/1